docs/en/reference/inheritance-mapping.rst

```diff
@@ -216,20 +246,21 @@ For Single-Table-Inheritance to work in scenarios where you are
 using either a legacy database schema or a self-written database
 schema you have to make sure that all columns that are not in the
 root entity but in any of the different sub-entities has to allow
-null values. Columns that have NOT NULL constraints have to be on
+null values. Columns that have ``NOT NULL`` constraints have to be on
 the root entity of the single-table inheritance hierarchy.
 
 Class Table Inheritance
 -----------------------
 
 `Class Table Inheritance <https://martinfowler.com/eaaCatalog/classTableInheritance.html>`_
 is an inheritance mapping strategy where each class in a hierarchy
-is mapped to several tables: its own table and the tables of all
+is mapped to several tables: Its own table and the tables of all
```

Comment:
I don't think that fix is correct: the second part isn't even a valid, standalone sentence (it has no verb). Even if it did, I would keep this lowercase.

Reply:
okay, will fix